CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 This contains only the most important and/or user-facing changes; for a full changelog, see the commit history.
 
+## [1.2.42](https://github.com/ably/ably-js/tree/1.2.42) (2023-07-24)
+
+- Auth: remain connected upon failed authorize unless returning explicit 403 [\#1385](https://github.com/ably/ably-js/pull/1385)
+- Make `Utils#inspectError` use `toString` for `Error`-like values [\#1391](https://github.com/ably/ably-js/pull/1391)
+- Rename some misnamed parameters [\#1392](https://github.com/ably/ably-js/pull/1392)
```

Comment:
This one probably doesn't need to go in the changelog since it was just an internal refactor?

Reply:
oopsie, will remove